scripts/kg_build.py

```
@@ -765,75 +1041,165 @@ def run(argv: Sequence[str] | None = None) -> dict[str, Any]:
+                run_id = _execute_pipeline(
+                    uri=uri,
+                    auth=auth,
+                    source_text=spec.text,
+                    database=args.database,
+                    embedder=embedder,
+                    llm=llm,
+                    splitter=splitter,
+                    reset_database=reset_pending,
+                )
+            except (OpenAIClientError, LLMGenerationError, EmbeddingsGenerationError) as exc:
+                raise RuntimeError("OpenAI request failed") from exc
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)